packages/vanchor-client/src/queries/RelayerFee.ts

```diff
@@ -20,14 +20,14 @@ export const GetVAnchorTotalRelayerFeeByChain = async (
   subgraphUrl: SubgraphUrl,
   vAnchorAddress: string
 ): Promise<TotalRelayerFeeByChain> => {
-  const query = `
-  query TotalRelayerFee {
-  vanchorTotalRelayerFee(id: "${vAnchorAddress.toLowerCase()}"){
+  const query = /* GraphQL */ `
```

Comment:
This `/* GraphQL */` thing is kinda pointless, why do you need this? Of course its a graphql query, the repo is called `webb-graphql`.

Reply:
This text is intended for implementing syntax highlighting in VSCode, aimed at detecting syntax errors in GraphQL queries. It is specifically recommended for GraphQL queries that are written using string template literals. For further details, please refer to this issue: [prettier/prettier#4360](https://github.com/prettier/prettier/issues/4360).